nuget/framework/nunit.nuspec

```diff
@@ -29,7 +29,10 @@ Supported platforms:
       <group targetFramework="net40" />
       <group targetFramework="net45" />
       <group targetFramework="netstandard1.6">
-        <dependency id="NETStandard.Library" version="1.6.0" />
+        <dependency id="NETStandard.Library" version="1.6.1" />
```

Comment:
What is effect of going from version 1.6.0 to 1.6.1 ?

Reply:
I don't know, but I do know that we are currently compiling against 1.6.1 so I made the nuspec match the csproj. Perhaps I should change the csproj.

Reply:
Actually, I doubt that these are supposed to show up in the .nuspec at all– https://andrewlock.net/what-is-the-netstandard-library-metapackage/